library/core/src/fmt/mod.rs

```diff
@@ -697,11 +697,21 @@ pub use macros::Debug;
 
 /// Format trait for an empty format, `{}`.
 ///
-/// `Display` is similar to [`Debug`], but `Display` is for user-facing
-/// output, and so cannot be derived.
+/// `Display` is similar to [`Debug`], but `Display` is for user-facing output,
+/// and so cannot be derived. When `Display` is implemented, the [`ToString`]
```

Comment:
```suggestion
/// `Display` is similar to [`Debug`], but `Display` is for user-facing output.
/// When `Display` is implemented, the [`ToString`]
```

As long as we're fixing this documentation: this conclusion doesn't actually automatically follow, and might not always be true.